packages/video_player/video_player/lib/src/web_vtt.dart

```diff
@@ -0,0 +1,197 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:html/dom.dart';
+
+import 'closed_caption_file.dart';
+import 'package:html/parser.dart' as html_parser;
+
+/// Represents a [ClosedCaptionFile], parsed from the WebVtt file format.
+// See: https://en.wikipedia.org/wiki/WebVtt
```

Comment:
Here seems also to be a public documentation for `WebVttCaptionFile`, we should use ///